packages/http-server/src/data-source.ts

```diff
@@ -62,7 +62,7 @@ export function createHTTPDataSource() {
               `Missing event handler on the server for event: ${JSON.stringify(message.event)}`
             )
           }
-          let res = await value.handler(payload)
+          let res = await value.handler(...payload)
```

Comment:
We now support an array of arguments to the handler, so components with multiple arguments (such as Tamagui ones) will work.